managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -3283,4 +3283,12 @@ public void setState(State state) {
     public ManagedLedgerConfig getConfig() {
         return config;
     }
+
+    /**
+     * check cursor reset status
+     * @return true if the cursor reset in progress
+     */
+    public boolean resetCursorInProgress() {
+        return RESET_CURSOR_IN_PROGRESS_UPDATER.get(this) == TRUE;
```

Comment:
`return RESET_CURSOR_IN_PROGRESS_UPDATER.get(this)`

Reply:
```
public static final int FALSE = 0;
    public static final int TRUE = 1;
    private static final AtomicIntegerFieldUpdater<ManagedCursorImpl> RESET_CURSOR_IN_PROGRESS_UPDATER =
        AtomicIntegerFieldUpdater.newUpdater(ManagedCursorImpl.class, "resetCursorInProgress");
    @SuppressWarnings("unused")
    private volatile int resetCursorInProgress = FALSE;
```

I submit a wrong code. Anyway, `resetCursorInProgress` is `int` type instead of `boolean`.